specification/dimensions/skupriceid.md

```diff
@@ -0,0 +1,54 @@
+### SKU Price ID
+
+A SKU Price ID is a unique identifier that defines the price of a SKU that incurred a charge. This identifier represents a unique collection of nullable attributes that determine the price. SKU Price ID can be referenced on a price list published by a provider to look up detailed information about the SKU and the unit price associated with the SKU. The composition of the detailed information associated with the price may differ from one provider to the next.
+
+Every SKU has one or more SKU Prices.  For providers that define their prices at the SKU level, the SKU ID and SKU Price ID shall be the same.
+
+The SKUPriceId column MUST be present in the billing data. This column MUST be of type String. The SKUId MUST NOT be null when the corresponding SKUId is not null. When SKUId is null, SKUPriceId MUST be null.
```

Comment:
Looks like the SKUID PR is referring to the column id as SkuId and not SKUId. Did you all come to an agreement on if its SKU or Sku when an acronym is used in the column name? Notice that we only do this for when we reference the column by ColumnID (in the normative paragraph only). Otherwise we refer to the 'thing' by Display Name.

Reply:
We did not... I wasn't even aware it was a topic of debate (my assumption was acronym = capitalized).  Between this comment and Michael's comment on #245, I see now that it's not resolved.

Reply:
I resolved the casing naming convention via a separate commit.  Resolving this conversation.